pwndbg/abi.py

```diff
@@ -136,6 +137,17 @@ def update():
 
         linux = 'Linux' in abi
 
+    if not linux:
+        msg = M.warn(
+            "The bare metal debugging will enable since the gdb's osabi is '%s' which is not 'GNU/Linux'.\n"
+            "Ex. the page resolving and memory de-referencing ONLY works on known pages.\n"
+            "This option is seleted from compile arguments of gdb client and will be corrected if you load an ELF that contains ABI information.\n"
```

Comment:
`This option is seleted from compile arguments of gdb client` -> `This option is based on gdb client compile arguments`

However maybe we should add also: `(as this specified the default osabi)` or sth like that - not sure in here.

Reply:
`load an ELF that contains ABI information` - this is also not clear what does it mean. Can we at least have a comment in this file about when or where the ELF contains it?

---
Below comment might be a different PR (or at least an issue pls).

We should probably start thinking about proper documentation for pwndbg (and have a `baremetal` section there).

Maybe it would be good to add `baremetal.md` file? @zachriggle how should we approach docs?

Reply:
Fixed the first one. I rewrited the description for the second one.